cmd/src/campaigns_common.go

```diff
@@ -176,8 +176,19 @@ func campaignsExecute(ctx context.Context, out *output.Output, svc *campaigns.Se
 	}
 	imageProgress.Complete()
 
+	pending = campaignsCreatePending(out, "Resolving repositories")
+	repos, err := svc.ResolveRepositories(ctx, campaignSpec)
+	if err != nil {
+		if _, ok := err.(campaigns.UnsupportedRepoSet); ok {
+			campaignsCompletePending(pending, "Resolved repositories. Found unsupported codehosts.")
```

Comment:
Should we also display the list of unsupported repositories? This feels exceptional enough that users will want as much detail as possible, generally speaking.

Reply:
Done! Happy to update the formatting of this in the future, but before I dive into the `output` lib right before branch cut, I think I'll go with the minimal but readable/copy-pastable version.

![screenshot_2020-09-15_13 44 17](https://user-images.githubusercontent.com/1185253/93206428-ab852080-f759-11ea-916b-cb1f8255aff1.gif)